lib/irule-tester

```diff
@@ -323,12 +374,12 @@ function should_select_pool {
   local reqUrl=$2
   local destPool=$3
 
-  local poolResp=$($curlCmd -A "$userAgent" $reqUrl -I | egrep -o "^Set-Cookie: BIGipServer${destPool}|Set-Cookie: LastSelectedPool=/[A-Za-z0-9_-]*/${destPool}" | dos2unix)
```

Comment:
Can you help understand what the significance of this block of change is?  I'm not seeing why this is better.